webserver/src/main/java/io/kestra/webserver/controllers/EditorController.java

```diff
@@ -0,0 +1,34 @@
+package io.kestra.webserver.controllers;
+
+import io.micronaut.http.MediaType;
+import io.micronaut.http.annotation.Controller;
+import io.micronaut.http.annotation.Get;
+import io.micronaut.scheduling.TaskExecutors;
+import io.micronaut.scheduling.annotation.ExecuteOn;
+import io.micronaut.validation.Validated;
+import io.swagger.v3.oas.annotations.Operation;
+import jakarta.inject.Inject;
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.util.List;
+import java.util.stream.Stream;
+
+@Slf4j
+@Validated
+@Controller("/api/v1/editors")
+public class EditorController {
+    @Inject NamespaceFileController namespaceFileController;
+    @Inject FlowController flowController;
```

Comment:
I would not use controller inside of controllers, this is kind of strange and breaks some clean architecture rules.
Controllers are here to deals with the HTTP layer, by directly calling it's method out of an HTTP context you may breaks things.

Better to use the repository here.